src/transformation/visitors/binary-expression/bit.ts

```diff
@@ -105,8 +106,9 @@ export function transformUnaryBitOperation(
     operator: lua.UnaryBitwiseOperator
 ): lua.Expression {
     switch (context.luaTarget) {
+        case LuaTarget.Universal:
         case LuaTarget.Lua51:
-            context.diagnostics.push(unsupportedForTarget(node, "Bitwise operations", LuaTarget.Lua51));
+            context.diagnostics.push(unsupportedForTarget(node, "Bitwise operations", context.luaTarget));
```

Comment:
For the other messages you left the version mentioned in the diagnostic to be 5.1 (which also makes sense). We should probably be consistent with this though.